### DISCUSSION TITLE
Other standards - input

### DISCUSSION THREAD

**Discussion by rzivny** [intuitem/ciso-assistant-community#24] [Ideas]
hi, is there a format description for other standards? I am preparing, for example, ISO/IEC 27017 and 18, I would like to put them in this tool and share them. Is this possible?

**Comment by Mohamed-Hacene** [comment]
Hi, we have a generic yaml format for all libraries, take a look inside our buit-in [libraries](https://github.com/intuitem/ciso-assistant-community/tree/main/library/libraries) and use the same architecture for the standard of your choice. To load it into your app, you can either upload it from the library view or put it directly into the libraries folder inside your source code (path is the same than the link I put before). 
If you have any trouble during the creation of your standard yaml file, feel free to ask us any questions you may have.
You can also join our [Discord](https://discord.gg/bUvEy6GD) if you need more instructions, I would be delighted to help you create your own library.

**Comment by eric-intuitem** [comment]
@rzivny we have pushed a tool to generate a library from an Excel file with a few examples. Please have a look to the tools/ directory.

**Comment by rzivny** [comment]
Thanks a lot,
 I will to test this one function. I can use on production version (3 month old) or use new complete version which we are rebuild?

Regards Radek

**Comment by eric-intuitem** [comment]
@rzivny definitely you need to upgrade to the new version and rebuild. We have made a huge step in improving the data model and library handling, we are confident this will provide sufficient expressiveness to avoid another major change. Ascendant compatibility will be managed once we reach v1.0.

**Comment by rzivny** [comment]
Hello,

Thaks a lot for new version. it looks really interesting. I would definitely like to contribute ideas regarding other functionalities. In relation to NIS2, I would like to test it as part of a package of security services such as SIEM, ThreatHunting and CISO management.

 

Regards

 

Radek
